lib/pages/host_scan_page.dart

```diff
@@ -18,16 +18,23 @@ class HostScanPage extends StatefulWidget {
 
 class _HostScanPageState extends State<HostScanPage>
     with TickerProviderStateMixin {
-  final Set<ActiveHost> _devices = {};
+  final Set<ActiveHost> _hosts = {};
   double _progress = 0;
   bool _isScanning = false;
   StreamSubscription<ActiveHost>? _streamSubscription;
+  late String? ip;
```

Comment:
Not clear about some parts in this page, like why is this not local variable.
Will check it out later as I am on the phone

Reply:
It should be private but not local, since I'm saving the result of NetworkInfo calls for latter use when giving name and icon to the device.

Reply:
Change `ip` and `gatewayIp` to be of type ActiveHost